helm/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v2
+name: geoserver
+version: 0.0.1
+appVersion: 0.0.1
```

Comment:
we need to think of a good way to manage this chart, many GeoServer  versions are supported so sticking to GeoServer version is incorrect